local-volume/provisioner/README.md

```diff
@@ -48,3 +48,18 @@ The basic components of the provisioner are as follows:
 - Controller: The controller runs a sync loop that coordinates the other components.
   The discovery and deleter run serially to simplify synchronization with the cache
   and create/delete operations.
+
+## Known Issues and Workarounds
```

Comment:
Can you actually add this to the top level README, which has all the known issues?

Reply:
Sure. Done.